db/schema.rb

```diff
@@ -310,7 +312,7 @@
     t.index ["organization_id"], name: "index_transfers_on_organization_id"
   end
 
-  create_table "users", id: :serial, force: :cascade do |t|
+  create_table "users", force: :cascade do |t|
```

Comment:
Weird, I don't know why these got changed. Let me know if I should switch 'em back.

Reply:
Don't sweat it -- it happens on my systems as well.

There appears to be disagreement between users editing in a MacOS environment and those editing in a linux environment, when migrations are run.  

Non-blocker. :+1: